src/emqx_mgmt.erl

```diff
@@ -605,23 +605,23 @@ import_auth_clientid(Lists) ->
     case ets:info(emqx_auth_clientid) of
         undefined -> ok;
         _ ->
-            [ mnesia:dirty_write({emqx_auth_clientid, ClientId, Password}) || #{<<"clientid">> := ClientId,
+            [ mnesia:dirty_write({emqx_auth_clientid, ClientId, base64:decode(Password)}) || #{<<"clientid">> := ClientId,
                                                                                <<"password">> := Password} <- Lists ]
     end.
 
 import_auth_username(Lists) ->
     case ets:info(emqx_auth_username) of
         undefined -> ok;
         _ ->
-            [ mnesia:dirty_write({emqx_auth_username, Username, Password}) || #{<<"username">> := Username,
+            [ mnesia:dirty_write({emqx_auth_username, Username, base64:decode(Password)}) || #{<<"username">> := Username,
                                                                                <<"password">> := Password} <- Lists ]
     end.
 
 import_auth_mnesia(Auths) ->
```

Comment:
We should remove `import_auth_mnesia/1, import_acl_mnesia/1, import_schemas/1`,  they are enterprise code, include the export parts.